### PR TITLE
New messages for pull rumors

### PR DESCRIPTION
**Pull request by DanielTavaresA**
To be able to use pull gossiping, we need new messages that will share the state of the received rumors per server.

### CODE REVIEW

File: protocol/query/method/rumor_state.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "method": {
+            "description": "[String] operation to be performed by the query",
+            "const": "rumor_state"
+        },
+
+        "params": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "rumor_state": {
```
Comment (sgueissa) [inline protocol/query/method/rumor_state.json:17]: you could just write state, we already know it is a rumor_state with the method

**Review by sgueissa — COMMENTED** [review]
great I would just suggest changing the name of the param to just state

**Review by sgueissa — APPROVED** [review]
all good for me now with the added answer

**Review by K1li4nL — APPROVED** [review]
Approving blindly

**Pull request merged**